backend/app/views/spree/admin/products/index.html.erb

```diff
@@ -75,6 +82,12 @@
             <td class="sku"><%= product.sku rescue '' %></td>
             <td class="status"><%= active_badge(available_status == Spree.t(:available), label: available_status) %></td>
             <td class="text-center"><%= display_price(product) %></td>
+            <td>
```

Comment:
@mad-eel let's skip this - we will have a site-wide store switcher https://spark-solutions.atlassian.net/browse/SD-1336